changelogs/fragments/3387-allocatable_ip.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway - allow to create instance with an allocatable IP (https://github.com/ansible-collections/community.general/pull/3418).
```

maybe it is a (lack of) terminology thing on my side but, here the description mentions the *allocatable IP*, whilst in the code itself it seems to be related to *public IP*. Assuming that they both refer to the same thing, I would suggest a tiny wordsmithing here, maybe:
```suggestion
  - scaleway - allow to create instance with a public allocatable IP (https://github.com/ansible-collections/community.general/pull/3418).
```
Would that make sense (and perhaps be clearer to someone not as familiar with Scaleway)?